src/main/java/no/brreg/conceptcatalogue/service/ConceptPublisher.java

```diff
@@ -35,15 +37,15 @@ public void send(String publisherId) {
     }
 
     public void sendNewDataSource(String publisherId, String harvestUrl) {
-        try {
-            ObjectNode payload = JsonNodeFactory.instance.objectNode();
+        ObjectNode payload = JsonNodeFactory.instance.objectNode();
```

Comment:
du har en enkelt fixup av kommit du nettop legg til... anbefaler å squashe de to commit